.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,28 @@
+name: Python package
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [2.7, 3.5, 3.6, 3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -e .
+    - name: Test with pytest
+      run: |
+        export KAGGLE_USERNAME=dinosaur
+        export KAGGLE_KEY=xxxxxxxxxxxx
```

Comment:
probably will fail for `401 - Unauthorized` as it is just an example authorizing not a real one...

Reply:
The actual kaggle secret token can be stored in the secrets section in the repository